docs/sources/setup-grafana/configure-security/configure-authentication/github.md

```diff
@@ -60,6 +60,10 @@ You may allow users to sign-up via GitHub authentication by setting the
 user successfully authenticating via GitHub authentication will be
 automatically signed up.
 
+You may also reference environment variables and local files in your GitHub auth configuration.
+The syntax for this is `$__<provider>{<argument>}` or `${<environment variable>}`.
+For more details refer to [Variable expansion](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#variable-expansion)
```

Comment:
Thanks for this reminder! I think we can note that variable expansion can be used here and just link to it - the syntax is already described there.

```suggestion
You can also use [variable expansion]({{< relref "../../configure-grafana/#variable-expansion" >}}) to reference environment variables and local files in your GitHub auth configuration.
```

Reply:
Good point!